addons/website_event/controllers/main.py

```diff
@@ -138,6 +139,7 @@ def dom_without(without):
             'pager': pager,
             'searches': searches,
             'search_path': "?%s" % werkzeug.url_encode(searches),
+            'events_cover_properties': [json.loads(c.cover_properties) for c in events],
```

Comment:
Don't load the cover_properties here, do it in your qweb template. So when you have an `event` variable in qweb, do:
```xml
<t t-set="cover_properties" t-value="json.loads(event.cover_properties)"/>
```
then use `cover_properties` (that way you only load when needed without having to search for the right already loaded cover properties set).